contracts/UFragmentsPolicy.sol

```diff
@@ -57,18 +55,25 @@ contract UFragmentsPolicy is Ownable {
     // The number of rebase cycles since inception
     uint256 public epoch;
 
-    uint256 private constant RATE_DECIMALS = 18;
-
-    uint256 private constant TARGET_RATE = 1 * 10**RATE_DECIMALS;
+    // TODO: Explain this parameter
+    uint256 public targetRate;
 
-    int256 private constant TARGET_RATE_SIGNED = int256(TARGET_RATE);
+    uint256 private constant RATE_DECIMALS = 18;
 
     // Due to the expression in computeSupplyDelta(), MAX_RATE * MAX_SUPPLY must fit into an int256.
     // Both are 18 decimals fixed point numbers.
     uint256 private constant MAX_RATE = 10**6 * 10**RATE_DECIMALS;
     // MAX_SUPPLY = MAX_INT256 / MAX_RATE
     uint256 private constant MAX_SUPPLY = ~(uint256(1) << 255) / MAX_RATE;
 
+    uint256 private constant CPI_DECIMALS = 6;
```

Comment:
I am leaning to have one variable DECIMALS=18 instead of RATE_DECIMALS/CPI_DECIMALS. Interested in hearing the counter argument.

Reply:
Makes sense, Having one `DECIMALS` variable makes the code cleaner. 

The only counter-argument is that this data from different oracles won't necessarily be in the same fixed-point format.